common/scripts/base.sh

```diff
@@ -139,3 +138,25 @@ function showUsageAndExitDefault() {
   echoBold "Ex: ./deploy.sh"
   exit 1
 }
+
+function deploy_common_service()
+{
+  if ! bash ${mesos_artifacts_home}/common/${1}/deploy.sh; then
+    echoError "Non-zero exit code returned when deploying ${1}"
+    exit 1
```

Comment:
@Thanu functions in base.sh only return values. It should not exit which would make it hard to troubleshoot. It should exit from script which is calling this function with a meaningful error msg.